### ISSUE TITLE
[UXW] Update the course details and topic descriptions on the Oppia.org site.

### ISSUE THREAD

**Issue by seanlip** [oppia/lessons-team#1540]
**Platform**
Oppia Web (though the descriptions are reused on Oppia Android).

**Describe the request**
Update the "intro text" for each classroom topic on the website. This was brought up by @stepmalt in a review of mocks from @juliafalarini and we decided to set up a UXW issue to update these so that they are clearer for learners.

**Information about deadlines/priorities**
This is a UXW project that can be done at the usual priority.

**DESIGN APPROVAL CHECKLIST -- DO NOT EDIT.**

- [ ] UX Writing approved
- [ ] Product approved

_For a design project to be completed, it needs approval from UXD, UXW, DS and PM. Please tag the following reviewers when your project is ready (at least one reviewer for each of the 4 bullet points below):_
- UXW: @stepmalt
- PM: @seanlip

**Comment by seanlip** [comment]
Linking to @stepmalt's doc [here](https://docs.google.com/document/d/1uF6rkt8qVYaSVxiUzXGskCkfEleJVO9ovvbHdHJg8As/edit#heading=h.vrszfdc8hvxb) and assigning this to @stepmalt.

**Comment by seanlip** [comment]
Hi @stepmalt -- just an update, I left a few comments on the doc. Could you please take a look? Thanks! https://docs.google.com/document/d/1uF6rkt8qVYaSVxiUzXGskCkfEleJVO9ovvbHdHJg8As/edit

**Comment by kmuir1991** [comment]
Hi @stepmalt and @seanlip ! Just wanted to check in on this one since it seems like a lot of work has been done on it so far, judging from the corresponding doc (which looks great, btw. Nice job, Stephanie!)

Is there a reason why the status is still "Not started"?

**Comment by stepmalt** [comment]
Hi Kate,

I didn’t see Sean’s email. Maybe it’s because I need to go reply to his
comments, which I will do tonight! I don’t know how this got buried.

Thank you for your feedback! So sorry I didn’t see this.

Warm regards,

Stephanie.

**Comment by kmuir1991** [comment]
Hi Stephanie,

Sounds good! Thanks for the update.

Best,
Kate
________________________________

**Comment by stepmalt** [comment]
Hi Kate, I have addressed Sean's comments and now I am unsure what the next step on this is. Does it go to you for finalization?

Let me know if there's anything more I can do regarding this!

https://docs.google.com/document/d/1uF6rkt8qVYaSVxiUzXGskCkfEleJVO9ovvbHdHJg8As/edit?usp=sharing

**Comment by seanlip** [comment]
@stepmalt Yes please get @kmuir1991's approval on UXW and then this can be implemented.  I also left a comment in the doc for you. Thanks!

**Comment by kmuir1991** [comment]
Hi @seanlip , just wanted to confirm that I approved this, so it should be good to implement.

**Comment by seanlip** [comment]
Thanks @kmuir1991 -- could you please update the status so it appears in the PM board? Please do this going forward for any projects that have been fully approved by UXD/UXW. Let me know if you have any problems with that, thanks!

**Comment by kmuir1991** [comment]
@seanlip I changed it to "Handed off to PM." Can't remember if that's the correct one or not?

**Comment by seanlip** [comment]
Yup that's right. Thanks! I'll process it in my weekly triage.

**Comment by stepmalt** [comment]
@seanlip Has this been sent off yet? Because I just noticed a small error as I was looking through it and omitted it on the document. But I'm unsure if it's too late. Sorry for any inconvenience.

**Comment by stepmalt** [comment]
@seanlip @kmuir1991 

Hi! I hope you're both doing well. This document was a few months old and upon looking back I noticed some ways I could add better word choice and phrasing to the text. Is there any way Kate could take one more look at it if it's not too late? If it is, no worries, I just thought I'd ask. I really want this to be a strong section.

I appreciate your time!

**Comment by seanlip** [comment]
@stepmalt This hasn't been sent out. Feel free to make further edits, I'll move it back to review status. Thanks!

**Comment by stepmalt** [comment]
@kmuir1991 Here is my work on the math topics. I edited, but didn't change a ton in the initial paragraph. I don't think it looks that much different from the copy already existing in the other project you wanted to pull this copy to. If you'd like, I can rewrite this and that one?

**Comment by stepmalt** [comment]
https://docs.google.com/document/d/1uF6rkt8qVYaSVxiUzXGskCkfEleJVO9ovvbHdHJg8As/edit?usp=sharing

Hi @kmuir1991 - can you take a look at this one final time before we send it off? I change a couple of things. Wasn't sure how you felt about the intro paragraph or if you wanted me to rewrite it. If all is good, we will send off. Sorry for you having to look at it again, I just noticed some edits I wanted to make a few weeks ago and it hadn't been sent off yet.

**Comment by kmuir1991** [comment]
Hi @stepmalt ! I just did another review and it all looks good to me :)

**Comment by seanlip** [comment]
Moving this to the lessons-team project board for execution. See https://docs.google.com/document/d/1uF6rkt8qVYaSVxiUzXGskCkfEleJVO9ovvbHdHJg8As/edit?usp=sharing

**Comment by stepmalt** [comment]
@seanlip can you move this back to getting UXW approval? I made some changed on it to improve it. Final iteration. I'd like the current UXW leads to take a final look at it and give last feedback. Thanks.

**Comment by stepmalt** [comment]
I can message the leads to look at it if that's ok? @seanlip

**Comment by seanlip** [comment]
Hi @stepmalt I think it's fine to leave it here, just ask the leads to approve in this thread and we can make the updates on the site once that's done. Thanks!

**Comment by tamiorendain** [comment]
Hi @seanlip, is this the same content as project #157? https://github.com/orgs/oppia/projects/7/views/22?pane=issue&itemId=83791882&issue=oppia%7Cdesign-team%7C157

If so, I think we should just use the updated copy we're working on in that project.

**Comment by seanlip** [comment]
Yes, closing this as a duplicate of oppia/design-team/#157. Thanks!
